test/sagemaker_tests/pytorch/training/integration/sagemaker/test_distributed_operations.py

```diff
@@ -66,41 +67,53 @@ def test_dist_operations_cpu(sagemaker_session, framework_version, ecr_image, in
 @pytest.mark.model("unknown_model")
 @pytest.mark.skip_cpu
 @pytest.mark.deploy_test
-def test_dist_operations_gpu(sagemaker_session, framework_version, instance_type, ecr_image, dist_gpu_backend):
+def test_dist_operations_gpu(sagemaker_session, n_virginia_sagemaker_session, framework_version, instance_type, ecr_image, n_virginia_ecr_image, dist_gpu_backend):
     """
     Test is run as multinode
     """
     instance_type = instance_type or 'ml.p2.xlarge'
-    _test_dist_operations(sagemaker_session, framework_version, ecr_image, instance_type, dist_gpu_backend)
+    _test_dist(sagemaker_session, n_virginia_sagemaker_session, framework_version, ecr_image, n_virginia_ecr_image, instance_type, dist_gpu_backend)
 
 
 @pytest.mark.processor("gpu")
 @pytest.mark.model("unknown_model")
 @pytest.mark.skip_cpu
-def test_dist_operations_multi_gpu(sagemaker_session, framework_version, ecr_image, dist_gpu_backend):
+def test_dist_operations_multi_gpu(sagemaker_session, n_virginia_sagemaker_session, framework_version, ecr_image, n_virginia_ecr_image, dist_gpu_backend):
     """
     Test is run as single node, but multi-gpu
     """
-    _test_dist_operations(sagemaker_session, framework_version, ecr_image, MULTI_GPU_INSTANCE, dist_gpu_backend, 1)
+    _test_dist(sagemaker_session, n_virginia_sagemaker_session, framework_version, ecr_image, n_virginia_ecr_image, MULTI_GPU_INSTANCE, dist_gpu_backend, 1)
 
 
 @pytest.mark.processor("gpu")
 @pytest.mark.integration("fastai")
 @pytest.mark.model("cifar")
 @pytest.mark.skip_cpu
 @pytest.mark.skip_py2_containers
-def test_dist_operations_fastai_gpu(sagemaker_session, framework_version, ecr_image):
+def test_dist_operations_fastai_gpu(sagemaker_session, n_virginia_sagemaker_session, framework_version, ecr_image, n_virginia_ecr_image):
     with timeout(minutes=DEFAULT_TIMEOUT):
-        pytorch = PyTorch(
-            entry_point='train_cifar.py',
-            source_dir=os.path.join(fastai_path, 'cifar'),
-            role='SageMakerRole',
-            instance_count=1,
-            instance_type=MULTI_GPU_INSTANCE,
-            sagemaker_session=sagemaker_session,
-            image_uri=ecr_image,
-            framework_version=framework_version,
-        )
+        estimator_parameter = {
+            'entry_point': 'train_cifar.py',
+            'source_dir': os.path.join(fastai_path, 'cifar'),
+            'role': 'SageMakerRole',
+            'instance_count': 1,
+            'instance_type': MULTI_GPU_INSTANCE,
+            'framework_version': framework_version,
+             
+        }
+        try:
+            pytorch = PyTorch(
+                sagemaker_session=sagemaker_session,
+                image_uri=ecr_image,
+                **estimator_parameter
+                )
+        except Exception as e:
```

Comment:
Exception should be raised for other exceptions

Reply:
Ack